diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
index 8fb94c5..3701ad0 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
@@ -1,17 +1,10 @@
 import React from 'react';
 
-import { Box, Button, Flex, Typography } from '@strapi/design-system';
-import { Link } from '@strapi/design-system/v2';
-import { ArrowRight } from '@strapi/icons';
+import { Box, Flex, Typography } from '@strapi/design-system';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
-import styled from 'styled-components';
 
-const WordWrap = styled(Typography)`
-  word-break: break-word;
-`;
-
-const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
+const HomeHeader = () => {
   const { formatMessage } = useIntl();
 
   return (
@@ -19,44 +12,12 @@ const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
       <Box paddingLeft={6} paddingBottom={10}>
         <Flex direction="column" alignItems="flex-start" gap={5}>
           <Typography as="h1" variant="alpha">
-            {hasCreatedContentType
-              ? formatMessage({
-                  id: 'app.components.HomePage.welcome.again',
-                  defaultMessage: 'Welcome 👋',
-                })
-              : formatMessage({
-                  id: 'app.components.HomePage.welcome',
-                  defaultMessage: 'Welcome on board!',
-                })}
-          </Typography>
-          <WordWrap textColor="neutral600" variant="epsilon">
-            {hasCreatedContentType
-              ? formatMessage({
-                  id: 'app.components.HomePage.welcomeBlock.content.again',
-                  defaultMessage:
-                    'We hope you are making progress on your project! Feel free to read the latest news about Strapi. We are giving our best to improve the product based on your feedback.',
-                })
-              : formatMessage({
-                  id: 'app.components.HomePage.welcomeBlock.content',
-                  defaultMessage:
-                    'Congrats! You are logged as the first administrator. To discover the powerful features provided by Strapi, we recommend you to create your first Content type!',
-                })}
-          </WordWrap>
-          {hasCreatedContentType ? (
-            <Link isExternal href="https://strapi.io/blog">
-              {formatMessage({
-                id: 'app.components.HomePage.button.blog',
-                defaultMessage: 'See more on the blog',
+            {
+              formatMessage({
+                id: 'app.components.HomePage.welcome',
+                defaultMessage: 'Welcome on board!',
               })}
-            </Link>
-          ) : (
-            <Button size="L" onClick={onCreateCT} endIcon={<ArrowRight />}>
-              {formatMessage({
-                id: 'app.components.HomePage.create',
-                defaultMessage: 'Create your first Content type',
-              })}
-            </Button>
-          )}
+          </Typography>
         </Flex>
       </Box>
     </div>
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
index a22f746..1c1f6c5 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
@@ -6,21 +6,17 @@
 import React, { useMemo } from 'react';
 
 import { Box, Grid, GridItem, Layout, Main } from '@strapi/design-system';
-import { LoadingIndicatorPage, useGuidedTour } from '@strapi/helper-plugin';
+import { LoadingIndicatorPage } from '@strapi/helper-plugin';
 import { Helmet } from 'react-helmet';
 import { FormattedMessage } from 'react-intl';
 import { useHistory } from 'react-router-dom';
 import styled from 'styled-components';
 
-import GuidedTourHomepage from '../../components/GuidedTour/Homepage';
-import isGuidedTourCompleted from '../../components/GuidedTour/utils/isGuidedTourCompleted';
 import { useContentTypes } from '../../hooks/useContentTypes';
 import { useEnterprise } from '../../hooks/useEnterprise';
 
 import cornerOrnamentPath from './assets/corner-ornament.svg';
-import ContentBlocks from './ContentBlocks';
 import HomeHeader from './HomeHeader';
-import SocialLinks from './SocialLinks';
 
 const LogoContainer = styled(Box)`
   position: absolute;
@@ -35,9 +31,6 @@ const LogoContainer = styled(Box)`
 export const HomePageCE = () => {
   // Temporary until we develop the menu API
   const { collectionTypes, singleTypes, isLoading: isLoadingForModels } = useContentTypes();
-  const { guidedTourState, isGuidedTourVisible, isSkipped } = useGuidedTour();
-  const showGuidedTour =
-    !isGuidedTourCompleted(guidedTourState) && isGuidedTourVisible && !isSkipped;
   const { push } = useHistory();
   const handleClick = (e) => {
     e.preventDefault();
@@ -75,14 +68,6 @@ export const HomePageCE = () => {
               />
             </GridItem>
           </Grid>
-          <Grid gap={6}>
-            <GridItem col={8} s={12}>
-              {showGuidedTour ? <GuidedTourHomepage /> : <ContentBlocks />}
-            </GridItem>
-            <GridItem col={4} s={12}>
-              <SocialLinks />
-            </GridItem>
-          </Grid>
         </Box>
       </Main>
     </Layout>
